docs/github-apps.md

```diff
@@ -100,7 +100,38 @@ That concludes the walkthrough!
 ### A Note on JWT Tokens
 Octokit.net aims to have no external dependencies, therefore we do not currently have the ability to generate/sign JWT tokens for you, and instead expect that you will pass in the appropriately signed JWT token required to authenticate the `GitHubApp`.
 
-Luckily one of our contributors [@adriangodong](https://github.com/adriangodong) has created a library `GitHubJwt` ( [GitHub](https://github.com/adriangodong/githubjwt) | [NuGet](https://www.nuget.org/packages/githubjwt) ) which you can use as per the following example.
+In order to create the token, you can create it manually using the following snippet.
+
+``` csharp
+var rsaPrivateKey = "..."; // RSA private key from the App configuration page
+var appId = 1; // The GitHub App Id
+
+using var rsa = RSA.Create();
```

Comment:
What's the advantage of the `using` statement here?

Reply:
Since the `RSA` object implements the `IDisposable` interface, its usually required to invoke the `.Dispose()` when the object leaves scope and is no longer needed. There could be some resources not managed by the runtime [that needs cleaning up](https://learn.microsoft.com/en-us/dotnet/api/system.idisposable?view=net-8.0). I haven't decompiled the code to see what's actually there. The syntax for declaring `using` was made available in [.NET 8](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/using).